
//require('assert').ok(false, "INTENSIONAL FAIL")

throw new Error ("INTENSIONAL ERROR")


